### PR TITLE
Initial-commit

### PR DESCRIPTION
**Pull request by 0sf**
I have updated the README.md I would love to hear what are the mistakes I made and correct them ASAP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,81 @@
...
+* [Contributing](#contributing)
+* [License](#license)
+
+## About the Project
+
+Make your own terminal aliases easily!
+
+### build with
+
+This project was built using JavaScript
```
Comment (shalithasuranga) [inline README.md]: Could you removed this section because anyone can see this from Github

File: README.md [hunk 2]
```diff
@@ -2,24 +2,81 @@
...
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+
+## About the Project
+
+Make your own terminal aliases easily!
```
Comment (shalithasuranga) [inline README.md]: Could you start with "Runn is command line tool which will make y..... etc"

File: README.md [hunk 3]
```diff
@@ -2,24 +2,81 @@
...
+## Getting Started
+
+### Prerequisites
+
+You need to install node package manager to runn this!
+
+* npm
+
+```sh
+npm install npm@latest -g
```
Comment (shalithasuranga) [inline README.md]: Could you add $ to start

File: README.md [hunk 4]
```diff
@@ -2,24 +2,81 @@
...
 $ runn cu
 ```
 
+## Roadmap
+
+See the [open issues](https://github.com/99xt/runn/issues) for a list of proposed features (and known issues).
+
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
```
Comment (shalithasuranga) [inline README.md]: maybe we can remove bold formatting ?

File: README.md [hunk 5]
```diff
@@ -2,24 +2,81 @@
...
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+
 ### License
 
-MIT
```
Comment (shalithasuranga) [inline README.md]: We can remove "See `LICENSE` for more information." and add link to License with MIT instead

**Review by shalithasuranga — CHANGES_REQUESTED** [review]
Thanks for the nice improvements. Could you please check inline comments and updated the PR

**Comment by 0sf** [comment]
@shalithasuranga Thank you for suggestions. I have corrected them with your guidelines. Please let me know if there are any mistakes. Thank you.

**Review by shalithasuranga — APPROVED** [review]
Nice work 👍

**Pull request merged**